Tasks/Common/sonarqube-common/sonarqube-common.ts

```diff
@@ -87,6 +90,28 @@ export function getSonarQubeEndpointFromInput(inputFieldName): SonarQubeEndpoint
     return new SonarQubeEndpoint(hostUrl, hostUsername, hostPassword);
 }
 
+// Returns, as an object, the contents of the 'report-task.txt' file created by SonarQube plugins
+// The returned object contains the following properties:
+//   projectKey, serverUrl, dashboardUrl, ceTaskId, ceTaskUrl
+export function getSonarQubeTaskReport(sonarPluginFolder: string) {
+    var reportFilePath:string = path.join(sonarPluginFolder, 'report-task.txt');
```

Comment:
Resolved, done.

It seemed better to hide the implementation of how the parsing is done, so I've moved that into a seperate method instead of the computing path stuff.